### PR TITLE
Car

### PR DESCRIPTION
**Pull request by Ryniek88**
Please check this out

### CODE REVIEW

File: src/Account.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+public class Account {
+    private int number;
+    private int balance;
+    private String name;
+    private String email;
+    private int phone;
+
+    public int getNumber() {
```
Comment (ghost) [inline src/Account.java:8]: Nice. You can use Lombok plugin to generate getters and setters next time. All you need to to is to add:
@Getter
@Setter
public class Account {

at the top of the class declaration. Very usefull, we us it in our company. Unfortunately you need to download the plugin in your Intelij(it's free)

File: src/Car.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    public boolean isConvertible() {
+        return convertible;
+    }
+
+    public void setMake(String make) {
+       if (make == null) make = "Unknown";
+       String lowercaseMake = make.toLowerCase();
+        switch (lowercaseMake) {
+            case "holden", "porsche", "tesla" -> this.make = make;
```
Comment (ghost) [inline src/Car.java:31]: You used braces in `default` code block but here you didn't. Please be consist.

File: src/Car.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    public void setDoors(int doors) {
+        this.doors = doors;
+    }
+
+    public void setConvertible(boolean convertible) {
+        this.convertible = convertible;
+    }
+
+    public void describeCar (){
+        System.out.println("Car " + make +" " + model + " " + color + " " + doors + " " + (convertible ? "Convertible" : "Unconvertible"));
```
Comment (ghost) [inline src/Car.java]: You can use more elegant solution using String.format() function:
`System.out.println(String.format("Car %s %s %s %d %s", make, model, color, doors, (convertible ? "Convertible" : "Unconvertible"));`

File: src/OOP.java [hunk 4]
```diff
@@ -4,5 +4,10 @@ public static void main(String[] args) {
         System.out.println(pot.high);
         System.out.println(pot.width);
         System.out.println(pot.color);
+        User user = new User (7,"Adam");
+        if (user.ifcanbuyalcohol()){
```
Comment (ghost) [inline src/OOP.java]: method name should be written in camelCase notation like `ifCanBuyAlcohol`

File: src/MainAccount.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+public class MainAccount {
+    public static void main(String[] args) {
+        Account account = new Account();
+        account.setBalance(1000);
+
+        account.withdrawing(400);
+
+        account.depositing(5000);
+
```
Comment (ghost) [inline src/MainAccount.java]: A lot of blank lines - please remove...

File: src/Car.java [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    public boolean isConvertible() {
+        return convertible;
+    }
+
+    public void setMake(String make) {
+       if (make == null) make = "Unknown";
+       String lowercaseMake = make.toLowerCase();
+        switch (lowercaseMake) {
```
Comment (ghost) [inline src/Car.java]: You can do `switch(make.toLowerCase()) {`
directly without creating new variable

**Review by Ryniek — CHANGES_REQUESTED** [review]
Please remove blank linkes

File: src/Car.java (hunk 2 above)
Comment (Ryniek88) [reply src/Car.java:31]: Done

File: src/Account.java (hunk 1 above)
Comment (Ryniek88) [reply src/Account.java:8]: I have already installed Lombok

File: src/Car.java (hunk 3 above)
Comment (Ryniek88) [reply src/Car.java]: You have one mistake: to less parenthesis

File: src/OOP.java (hunk 4 above)
Comment (Ryniek88) [reply src/OOP.java]: Done

File: src/MainAccount.java (hunk 5 above)
Comment (Ryniek88) [reply src/MainAccount.java]: Done

File: src/Car.java (hunk 2 above)
Comment (Ryniek88) [reply src/Car.java:31]: When I used braces my cod dosen't work propely. I shouldn't do this

File: src/Car.java (hunk 6 above)
Comment (Ryniek88) [reply src/Car.java]: Done

File: src/Car.java (hunk 2 above)
Comment (ghost) [reply src/Car.java:31]: This is a feature introduced in Java 12 known as "Switch Expressions". What is your Java version configured in Intelij? File > Project Structure > SDK. You should work on Java 17(LTS version - Long term support) or even 21(the latest version)

**Review by Ryniek — APPROVED** [review]

File: src/Car.java (hunk 3 above)
Comment (ghost) [reply src/Car.java]: Right, sorry

File: src/Car.java (hunk 2 above)
Comment (Ryniek88) [reply src/Car.java:31]: java 17.0.6 is my version of Java

**Pull request merged**